app/js/components/orderEntry/RenderOrderType.jsx

```diff
@@ -19,6 +19,9 @@ const RenderOrderType = (props) => {
         />
       );
     }
+    case orderTypes.ORDER_FROM_SETS.id: {
+      return "<InsertComponent />";
```

Comment:
WHy is the name of the component `InsertComponent`? Is there any particular reason?

Reply:
It is just a placeholder for now, when the new component is created it would be replaced

Reply:
Or do you just mean to place it there after the component is ready?

Reply:
Ok, great